pdns/recursordist/test-syncres_cc.cc

```diff
@@ -438,12 +438,18 @@ int genericDSAndDNSKEYHandler(LWResult* res, const DNSName& domain, DNSName auth
         /* sign the SOA */
         addRRSIG(keys, res->d_records, auth, 300, false, boost::none, boost::none, now);
         /* add a NSEC denying the DS */
-        std::set<uint16_t> types = {QType::NSEC};
+        std::set<uint16_t> types = { nsec3 ? QType::NSEC : QType::NSEC3 };
```

Comment:
check formatting (done on the recursordist file) complains:
```suggestion
        std::set<uint16_t> types = {nsec3 ? QType::NSEC : QType::NSEC3};
```